lib/core/src/lib/notifications/components/notification-history.component.spec.ts

```diff
@@ -83,12 +85,12 @@ describe('Notification History Component', () => {
             fixture.detectChanges();
             fixture.whenStable().then(() => {
                 fixture.detectChanges();
-                expect(component.notifications.length).toBe(1);
+                expect(component.unreadNotifications.length).toBe(1);
                 const markAllAsRead = overlayContainerElement.querySelector<HTMLButtonElement>('#adf-notification-history-mark-as-read');
-                markAllAsRead.click();
+                markAllAsRead?.click();
                 fixture.detectChanges();
-                expect(storage.getItem(NotificationHistoryComponent.NOTIFICATION_STORAGE)).toBeNull();
-                expect(component.notifications.length).toBe(0);
+                expect(component.unreadNotifications).toEqual([]);
```

Comment:
This expect is redundant because below you test if length is 0 so if length is 0 then that means array is [].

Reply:
removed redundant expect clauses.